app/src/main/java/com/github/pockethub/ui/repo/RepositoryPagerAdapter.java

```diff
@@ -66,40 +66,44 @@ public RepositoryPagerAdapter(AppCompatActivity activity,
     @Override
     public CharSequence getPageTitle(int position) {
         switch (position) {
-        case 0:
-            return resources.getString(R.string.tab_news);
-        case 1:
-            return resources.getString(R.string.tab_code);
-        case 2:
-            return resources.getString(R.string.tab_commits);
-        case 3:
-            return resources.getString(R.string.tab_issues);
-        default:
-            return null;
+            case 0:
+                return resources.getString(R.string.tab_readme);
+            case 1:
+                return resources.getString(R.string.tab_news);
+            case 2:
+                return resources.getString(R.string.tab_code);
+            case 3:
+                return resources.getString(R.string.tab_commits);
+            case 4:
+                return resources.getString(R.string.tab_issues);
+            default:
+                return null;
         }
     }
 
     @Override
     public Fragment getItem(int position) {
         switch (position) {
-        case 0:
-            return new RepositoryNewsFragment();
-        case 1:
-            codeFragment = new RepositoryCodeFragment();
-            return codeFragment;
-        case 2:
-            commitsFragment = new CommitListFragment();
-            return commitsFragment;
-        case 3:
-            return new IssuesFragment();
-        default:
-            return null;
+            case 0:
+                return new RepositoryReadmeFragment();
```

Comment:
Didn't think about that. As is there is just a white page so the choices are

1. Remove README tab.
2. Add a text saying there is no README.

Preference?